front/src/modules/object-record/components/RecordTableContainerMockMode.tsx

```diff
@@ -0,0 +1,79 @@
+import styled from '@emotion/styled';
```

Comment:
I feel we should not re-add this to object-record folder and create a separated company-record-table-mock module